src/model/ki2600.rs

```diff
@@ -184,7 +198,11 @@ impl NonBlock for Instrument {
 
 impl Drop for Instrument {
     fn drop(&mut self) {
-        let _ = self.interface.write_all(b"password\n");
```

Comment:
I think this "password" write was there to logout the instrument !

Reply:
You are correct. I'll add it back in.